samples/remote_attestation/enclave_a/ecalls.cpp

```diff
@@ -56,6 +56,25 @@ int verify_report_and_set_pubkey(
         pem_key, key_size, remote_report, remote_report_size);
 }
 
+/**
+ * Create ephemeral symmetric key, encrypt using other enclave's public key
+ * and sign using private key
+ */
+int establish_secure_channel(uint8_t** data, size_t* size)
+{
+    return dispatcher.establish_secure_channel(data, size);
+}
+
+/**
+ * Verify signature, decrypt using private ke
```

Comment:
typo "ke" at end

Reply:
Saw this and fixed it at my end already. Thanks for flagging all the places that I copied this line over.